modules/rostests/rosautotest/CWineTest.cpp

```diff
@@ -64,11 +64,27 @@ CWineTest::GetNextFile()
     WIN32_FIND_DATAW fd;
 
     /* Did we already begin searching for files? */
-    if(m_hFind)
+    if (m_hFind)
     {
         /* Then get the next file (if any) */
-        if(FindNextFileW(m_hFind, &fd))
-            FoundFile = true;
+        if (FindNextFileW(m_hFind, &fd))
+        {
+            // printf("cFileName is '%S'.\n", fd.cFileName);
+            /* if it was NOT rosautotest.exe then proceed as normal */
+            if (wcsicmp(fd.cFileName, TestName) != 0)
```

Comment:
```suggestion
            if (_wcsicmp(fd.cFileName, TestName) != 0)
```
Using the conformant name.